engine/ivy.xml

```diff
@@ -102,6 +102,7 @@
 	<dependency org="org.eclipse.jetty" name="jetty-servlet" rev="8.1.15.v20140411" transitive="false"  conf="default->default"/>
 	<dependency org="org.eclipse.jetty" name="jetty-xml" rev="8.1.15.v20140411" transitive="false"  conf="default->default"/>
 	<dependency org="org.eclipse.jetty" name="jetty-webapp" rev="8.1.15.v20140411" transitive="false"  conf="default->default"/>
+	<dependency org="com.googlecode.log4jdbc" name="log4jdbc" rev="1.2" transitive="false" conf="default->default"/>
```

Comment:
We cannot add new dependencies without following the new component process. Please see Marc to discuss.